<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameScene</class>
 <widget class="QWidget" name="GameScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VolleyBall</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recources.qrc">
    <normaloff>:/images/icon.ico</normaloff>:/images/icon.ico</iconset>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>650</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/ship1020.png);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startNewGame">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Start new Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pauseGame">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>0</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/pauseimage.png);
background-repeat: no-repeat;
border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="scorePlayer2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>40</y>
     <width>466</width>
     <height>89</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="scorePlayer1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>467</width>
     <height>89</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="winLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>601</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>scorePlayer2</zorder>
  <zorder>winLabel</zorder>
  <zorder>scorePlayer1</zorder>
  <zorder>pauseGame</zorder>
  <zorder>startNewGame</zorder>
 </widget>
 <resources>
  <include location="recources.qrc"/>
 </resources>
 <connections/>
</ui>
